extensions/adobe/experience/analytics/context-data.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "context-data",
+  "type": "object",
+  "description": "A free form field where Analytics context data will be captured.",
+  "definitions": {
```

Comment:
This definition should also have a "properties" object which will further have the map object you want to define.
Example:
`  "definitions": {
    "context-data-datatype": {
      "properties": {
        "xdm:<Map Object Name>": {
          "type": "object",
          "meta:xdmType": "map",
          "additionalProperties": {
            "type": "string",
			"description": "Your Description"
          }
        }
      }
    }
  }
`
an existing example you can refer is 
https://github.com/adobe/xdm/blob/master/components/fieldgroups/shared/identitymap.schema.json

Please make the changes.

Reply:
thank you! I'll add the changes!